core/aws-core/src/main/java/software/amazon/awssdk/awscore/internal/AwsErrorCode.java

```diff
@@ -64,6 +64,7 @@
         retryableErrorCodes.add("PriorRequestNotComplete");
         retryableErrorCodes.add("RequestTimeout");
         retryableErrorCodes.add("RequestTimeoutException");
+        retryableErrorCodes.add("InternalError");
```

Comment:
Nit: Looks like we used to specify initial capacity size here, but that is probably an over-optimization.